control-plane/stor-port/src/types/v0/store/node.rs

```diff
@@ -417,9 +427,27 @@ impl NodeSpec {
 
 impl From<NodeSpec> for models::NodeSpec {
     fn from(src: NodeSpec) -> Self {
+        let labels = if src.labels.is_empty() {
+            None
+        } else {
+            Some(
+                src.labels
+                    .iter()
+                    .filter_map(|(key, value)| {
+                        if value.is_empty() {
+                            None
+                        } else {
+                            Some(format!("{}={}", key, value))
+                        }
+                    })
+                    .collect::<Vec<_>>()
+                    .join(";"),
```

Comment:
hmm, should we change labels in NodeSpec to be a map? It would also be the same as pool labels, so it'd be consistent
CC @Abhinandan-Purkait

Reply:
Its a map only